sdk/core/azure-core/azure/core/polling/base_polling.py

```diff
@@ -447,6 +448,30 @@ def initialize(self, client, initial_response, deserialization_callback):
         except OperationFailed as err:
             raise HttpResponseError(response=initial_response.http_response, error=err)
 
+    def get_continuation_token(self):
+        # type() -> str
+        import pickle
+        return base64.b64encode(pickle.dumps(self._initial_response)).decode('ascii')
+
+    @classmethod
+    def from_continuation_token(cls, continuation_token, **kwargs):
+        # type(str, Any) -> Tuple
+        try:
+            client = kwargs["client"]
+        except KeyError:
+            raise ValueError("Need kwarg 'client' to be recreated from continuation_token")
+
+        try:
+            deserialization_callback = kwargs["deserialization_callback"]
+        except KeyError:
+            raise ValueError("Need kwarg 'deserialization_callback' to be recreated from continuation_token")
+
+        import pickle
```

Comment:
Is importing pickle really slow? Just wondering why we've not added this at the top

Reply:
For thing that we'll use only 0.1% of the time, I prefer a local import, CLI is already fighting enough with import perf